pyriemann/utils/viz.py

```diff
@@ -109,29 +113,46 @@ def plot_cospectra(cosp, freqs, *, ylabels=None, title="Cospectra"):
     return fig
 
 
-def plot_waveforms(X, display, *, time=None, **kwargs):
+def plot_waveforms(X, display, *, times=None, **kwargs):
```

Comment:
reading the code I see that using kwargs is really not necessary here.

when you do things like:

`n_bins = kwargs.get('n_bins', 50)`

just put a parameter in the function signature with `n_bins=50`

it makes the documentation of such functions so much easier.